### PR TITLE
Change the gRPC API for visualising a graph to stream the resulting vdag

### PR DESCRIPTION
**Pull request by sebtomba**
## Overview
Change the gRPC API for visualising a graph to stream the resulting vdag

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-3118

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### CODE REVIEW

File: graphz/src/main/scala/coop/rchain/graphz/Graphz.scala [hunk 1]
```diff
@@ -1,11 +1,13 @@
 package coop.rchain.graphz
 
+import java.io.FileOutputStream
+
+import scala.collection.mutable.ListBuffer
```
Comment (lukasz-golebiewski) [inline graphz/src/main/scala/coop/rchain/graphz/Graphz.scala]: What's the motivation behind using a mutable collection inside the `MonadState`?

Comment (sebtomba) [reply graphz/src/main/scala/coop/rchain/graphz/Graphz.scala]: @lukasz-golebiewski The reason for a mutable `ListBuffer` is that we want to append and not prepend lines of strings. Of course, we could prepend and then reverse the whole list later. But this seems error-prone to me.

Comment (lukasz-golebiewski) [reply graphz/src/main/scala/coop/rchain/graphz/Graphz.scala]: Why not use `Vector`?

Comment (sebtomba) [reply graphz/src/main/scala/coop/rchain/graphz/Graphz.scala]: @lukasz-golebiewski np

**Pull request merged**